tests/functional/features/healthcare_worker_retrieve.feature

```diff
@@ -0,0 +1,117 @@
+        And the response body does not contain id
+        And the response body is the Empty Authorization header response
+
+    Scenario: Attempt to retrieve a patient with an invalid authorization header
+        Given I am a healthcare worker
+        And I have a header Authorization value of "Bearer abcdef123456789"
+
+        When I retrieve a patient
+
+        Then I get a 401 HTTP response code
+        And the X-Request-ID response header matches the request
+        And the X-Correlation-ID response header matches the request
+        And the response body does not contain id
+        And the response body is the Invalid Access Token response
+
+    Scenario: Attempt to retrieve a patient without stating a role
+        Given I am a healthcare worker
+        And I don't have a NHSD-Session-URID header
+
+        When I retrieve a patient
+
+        Then I get a 400 HTTP response code
+        And the X-Request-ID response header matches the request
+        And the X-Correlation-ID response header matches the request
+        And the response body does not contain id
+        And the response body is the Missing URID header response
+
+    Scenario: Attempt to retrieve a patient with an invalid role
+        Given I am a healthcare worker
+        And I have a header NHSD-Session-URID value of "invalid"
+
+        When I retrieve a patient
+
+        Then I get a 400 HTTP response code
+        And the X-Request-ID response header matches the request
+        And the X-Correlation-ID response header matches the request
+        And the response body does not contain id
+        And the response body is the Invalid URID header response
+
+    Scenario: Attempt to retrieve a patient without Request ID header
+        Given I am a healthcare worker
+        And I have an empty X-Request-ID header
+
+        When I retrieve a patient
+
+        Then I get a 400 HTTP response code
+        And the response header does not contain X-Request-ID
+        And the X-Correlation-ID response header matches the request
+        And the response body does not contain id
+        And the response body is the Empty X-Request ID response
+
+    Scenario: Attempt to retrieve a patient with an invalid X-Request-ID
+        Given I am a healthcare worker
+        And I have a header X-Request-ID value of "1234"
+
+        When I retrieve a patient
+
+        Then I get a 400 HTTP response code
+        And the X-Request-ID response header matches the request
+        And the X-Correlation-ID response header matches the request
+        And the response body does not contain id
+        And the response body is the Invalid X-Request ID response
+
+    Scenario: Attempt to retrieve a patient with a missing X-Request-ID
+        Given I am a healthcare worker
+        And I don't have a X-Request-ID header
+
+        When I retrieve a patient
+
+        Then I get a 412 HTTP response code
```

Comment:
why a 412 not a 400? Not saying it 's wrong

Reply:
The [original test](https://github.com/NHSDigital/personal-demographics-service-api/blob/a227be578551cb6baa381fab408eb0e4b8509086/tests/user_restricted/test_user_restricted.py#L141) had it as a 412. I didn't question it.

Looking at the [docs](https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir#get-/Patient/-id-)'s 4XX responses: 412 is not listed; and `<html>
<body>
<!--StartFragment-->

400 | MISSING_VALUE | Missing header parameter. For details, see the diagnostics field.
-- | -- | --


<!--EndFragment-->
</body>
</html>`

may be more appropriate. If in agreement, I'll raise a separate ticket for this.